src/main/resources/org/jenkins/plugins/lockableresources/actions/LockableResourcesRootAction/tableQueue/table.jelly

```diff
@@ -49,7 +49,11 @@ THE SOFTWARE.
       </j:when>
       <j:otherwise>
         <j:if test="${oldestQueue.takeTooLong()}">
-          <script type="text/javascript">notificationBar.show('${%queue.warning.count(queue.getAll().size(), h.getTimeSpanString(oldestQueue.getQueuedTimestamp()))}', notificationBar.WARNING)</script>
+          <span class="lockable-resources-queue-too-long-message"
+                data-warning-message="${%queue.warning.count(queue.getAll().size(), h.getTimeSpanString(oldestQueue.getQueuedTimestamp()))}"
+                style="display:none"/>
```

Comment:
Could you please give me your opinion about two issues:

1. In https://github.com/jenkinsci/config-file-provider-plugin/pull/342 we're using an invisible entry rather than `display: none`. Which is preferable and why? If they are more or less the same, then I don't have a strong preference which one we follow, but I do prefer that this project's PRs be consistent with each other. That way, we are less likely to have strange bugs caused by inconsistent implementation strategies.
2. In https://github.com/jenkinsci/config-file-provider-plugin/pull/342 we aren't putting a class attribute but instead using `const contentTypeElement = document.querySelector("span[data-content-type]")`. Similar question as to which is preferable, and if we had to pick a project-wide convention, which would it be?

Reply:
1. For me everything that's inside the `form` tag lib semantically belongs to forms. We do not do anything with forms here, we only want something to hold data that we use in scripts. So if we speak only about holding data I'd stick to spans.
2. I think we should use classes and stay away from sometimes overcomplicated selectors. Class selectors are concise understandable by everyone. In case `js` files live far away from `jelly` ones having a class name it is trivially easy to find all parts of the equation. With classes we are better protected against potential future markup changes. Someone may introduce new element that may be unintentionally targeted by those fancy selectors. It is less likely that someone will add a very specific class name to their element unless they know what they're doing.

Reply:
So regarding 1), your preference would be to use a `span` with `display: none` when there is a form and a `span` without `display: none` inside of an `invisibleEntry` when there is not a form? Or would you want to use a `span` with `display: none` with or without a form? Or am I misunderstanding and your preference is for something else entirely?

Regarding 2), I agree with the argument about avoiding fragility.

Reply:
I wouldn't use `invisibleEntry` at all for the sake of holding data for JavaScript. 

From its doc:
> Invisible `<f:entry>` type. Useful for adding hidden field values.

Would use it that same way `entry` is used, but when I don't want the field to be visible. E.g. hold a CSRF key in a form that has to be submitted with the form, but we don't want anyone to tamper with it.